uml/sequence_diagrams/Map.txt

```diff
@@ -0,0 +1,65 @@
+@startuml
+participant DisplayViews 
+participant MapView
+participant GPS
+participant Bluetooth
+participant DisplayError 
+
+opt Map
+  DisplayViews --> MapView : GoTo Map
+end
+opt Views
+  MapView --> DisplayViews : GoTo Views

```

Comment:
For consistency, would you be happy to move that into the event loop in the same way you did for `ClockView.png` ?

It made more sense for me to have the exit transition in the event loop

Reply:
fixed